### PR TITLE
feat(AAiT-backend-2b): Set up project files

### PR DESCRIPTION
**Pull request by Amanuel94**

### CODE REVIEW

File: aait/backend/group-2B/Domain/Common/BaseAuditableEntity.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace SocialSync.Domain.Common;
+
```
Comment (samiabat) [inline aait/backend/group-2B/Domain/Common/BaseAuditableEntity.cs:2]: why do you add this entity? You can add the fields to the BaseEntity.cs.

File: aait/backend/group-2B/.vscode/settings.json
Comment (YabTek) [inline aait/backend/group-2B/.vscode/settings.json:1]: It is better to remove this file whenever you make a commit.

File: aait/backend/group-2B/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (samiabat) [inline aait/backend/group-2B/.vscode/settings.json:1]: this kind of file should not be tracked. add to .gitignore

File: aait/backend/group-2B/Persistence/Migrations/20230822131250_initial-migration.Designer.cs
Comment (YabTek) [inline aait/backend/group-2B/Persistence/Migrations/20230822131250_initial-migration.Designer.cs:1]: Also this migration folder should not be pushed.

**Review by YabTek — COMMENTED** [review]
Since this is a project setup, it will be good to add the test folder and complete the application layer.

File: aait/backend/group-2B/Domain/Common/BaseAuditableEntity.cs (hunk 1 above)
Comment (ffekirnew) [reply aait/backend/group-2B/Domain/Common/BaseAuditableEntity.cs]: Noted. The file has been removed and consolidated into one.

File: aait/backend/group-2B/.vscode/settings.json (hunk 2 above)
Comment (ffekirnew) [reply aait/backend/group-2B/.vscode/settings.json]: Noted. All sensitive files have been gitignored.

File: aait/backend/group-2B/.vscode/settings.json
Comment (ffekirnew) [reply aait/backend/group-2B/.vscode/settings.json:1]: Noted. All sensitive files have been gitignored.

File: aait/backend/group-2B/Persistence/Migrations/20230822131250_initial-migration.Designer.cs
Comment (ffekirnew) [reply aait/backend/group-2B/Persistence/Migrations/20230822131250_initial-migration.Designer.cs:1]: Noted. All sensitive files have been gitignored.

**Review by yonass08 — APPROVED** [review]
Looks good.

**Review by bahailu-abera — APPROVED** [review]

**Pull request merged**